src/SprykerEco/Zed/Easycredit/Business/Api/Client/EasycreditClient.php

```diff
@@ -1,14 +0,0 @@
-<?php
-
-/**
- * MIT License
- * For full license information, please view the LICENSE file that was distributed with this source code.
- */
-
-namespace SprykerEco\Zed\Easycredit\Business\Api\Client;
-
-use GuzzleHttp\Client;
-
-class EasycreditClient extends Client
-{
-}
```

Comment:
Can we really remove this?

Reply:
I removed it because it throws an phpstan error and it has no usage found by the IDE:
The GuzzleClient has the final class annotation and therfore cant be extended.

Reply:
To still have an client I added the dependency and injected it in the factory